src/predictions/profiles_mlcorelib/ml_core/preprocess_and_predict.py

```
@@ -346,11 +346,20 @@ def predict_scores_rs(df: pd.DataFrame) -> pd.DataFrame:
 
     model_path = os.path.join(output_dir, args.json_output_filename)
 
+    inputs_info: List[utils.InputsConfig] = []
+    try:
+        if isinstance(args.inputs, list) and isinstance(args.inputs[0], dict):
```

What would be the else condition? If we expect it should always go to the if condition, let's skip the condition? And let it error out in case of unexpected behavior? Currently it will silently make mistakes if the instance checks fail and that's not intended.